src/cmds/scripts/pbs_server

```diff
@@ -39,6 +39,11 @@
 # Source the file that sets PGSQL_LIBSTR
 . "$PBS_EXEC"/libexec/pbs_pgsql_env.sh
 
+
+# Loading dynamic library at PBS_EXEC/lib
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$PBS_EXEC/lib/
```

Comment:
This will be lib64 for SUSE platforms, so you need to check for both. Something similar is done for PostgreSQL in src/cmds/scripts/pbs_pgsql_env.sh